### ISSUE TITLE
app_id is deprecated

### ISSUE THREAD

**Issue by kickbk** [paypal/merchant-sdk-ruby#39]
To other developers, please be aware that app_id is not needed for EC dialogs. We implemented recurring payment without it and tested on live. You can just remove that line from your paypal.yml.

PayPal needs to update their [documentation](https://developer.paypal.com/docs/classic/api/apiCredentials/).

You do need your api credentials of course (username password and signature) to go live. Lik above explains how.

Previously you would need to request a classic app to be approved via [this dashboard page](https://developer.paypal.com/developer/applications/), but when clicking on "Manage NVP/SOAP API apps" you will notice they plan to deprecate it and there is no "New App" button to create one.

I suppose the gem should be updated so it doesn't generates an app_id and some documentation could be helpful. Nonetheless, thanks for maintaining the gem. It's certainly very helpful and works well.

**Comment by pp-randy** [comment]
Yeah that is a good point, I don't think anything in this SDK / gem requires the App IDs.

**Comment by pp-randy** [comment]
For new integrations though I would steer folks to the REST API, or a Braintree option, as this SDK is only in maintenance mode.   

The REST API SDK is here https://github.com/paypal/PayPal-Ruby-SDK.  We will be having a new version of that REST SDK coming in the future.
